docs/license.rst

```diff
@@ -1,4 +1,4 @@
 License
```

Comment:
Not sure about my knowledge on this, but do RST titles need to be framed with equal signs at the top AND at the bottom?
```
=======
License
=======
```

Also, I realize that my suggestion is NOT part of your PR, so feel free to ignore it.

Reply:
Functionally no, but would be nice to be more consistent, and we have this mixed throughout. RST is quite flexible with marking headers.

Reply:
Cool! Thanks for letting me know. :)